namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -23,6 +23,7 @@ class ConsulDtabStore(
   datacenter: Option[String] = None,
   readConsistency: Option[ConsistencyMode] = None,
   writeConsistency: Option[ConsistencyMode] = None,
+  overrideHandlerUrl: Option[String] = None,
```

Comment:
Maybe just require the handler url to be explicitly passed in, rather than passing in an override.

Reply:
Yea that could work, but I wanted the default cause to always to be the `storage/....` endpoint optionally set the URL if we wanted it changed. I guess since the `ConsulDtabStore` isn't used anywhere else it'd be fine to make it more explicit.